src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -56,8 +56,10 @@ public void storeFood(FoodItem food) {
   /** Stores Meal in Datastore. */
   public void storeMeal(EatenMeal meal) {
     Entity mealEntity = new Entity("UserMeal", meal.getID().toString());
-    mealEntity.setProperty("Foods", meal.getFoods());
-    mealEntity.setProperty("Amounts", meal.getAmounts());
+    mealEntity.setProperty("Food", meal.getFood());
+    mealEntity.setProperty("UserId", meal.getUserId());
+    mealEntity.setProperty("Amount", meal.getAmount());
+    mealEntity.setProperty("MealType", meal.getMealType());
```

Comment:
Do you know how Datastore handles enums like this?

Reply:
I did some quick googling and it seems like it's fine when storing, but when querying you need to declare parameters. It seems kind of complicated, so I think i'm going to use the ordinal method to assign integer values to the enums and store that int instead